packages/aragon-wrapper/src/core/proxy/index.js

```diff
@@ -12,12 +12,16 @@ export default class Proxy {
   }
 
   // TODO: Make this a hot observable
-  events (eventNames) {
+  events (eventNames, fromBlock) {
```

Comment:
```suggestion
  events (eventNames, fromBlock = this.initializationBlock) {
```

I am unsure whether this is allowed :D

Reply:
I wasn't sure. But it is: https://gist.github.com/2color/316ab17bcf1f6aa064de73653bc9df45